src/features/dialog-manager/reducer.js

```diff
@@ -43,6 +37,12 @@ const initialState = {
         min_intelligence: 0,
         min_charisma: 0,
     },
+    characterCreation: false,
+    character: {
+        characterName: 'Adventurer',
```

Comment:
Is this what we want the default name of the player to be? Will we want this to be a default in the input field for this as well?

Reply:
The default can be changed to whatever is preferred, I merely put that in because it made sense to me. I'm unsure about making it a default in the input field.